opdracht-1.js

```diff
@@ -0,0 +1,55 @@
+//Uitwerkingen opdracht 1
+
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+//1. Maak een variabele aan waarin alle tv typen worden opgeslagen;
+//2. Gebruik de map-methode om de array te doorzoeken;
+//3. Return de waarden: namen van tv's;
+//4. Console log de variabele.
+
+const typeOfTv = inventory.map((tvName) => {
+    return tvName.name;
+})
+
+console.log(typeOfTv);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+//1. Maak een variabele aan waarin de uitverkochte tv's worden opgeslagen;
+//2. Gebruik de filter-methode om de array te doorzoeken;
+//3. Geef de zoekcriteria mee: originalStock === sold;
+//4. Return de objecten;
+//5. Console log de variabele.
+
+const tvOutOfStock = inventory.filter((soldOut) => {
+    return soldOut.originalStock === soldOut.sold
+})
+
+console.log(tvOutOfStock);
+
+
+//Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+
+//1. Maak een variabele aan waarin de tv's met Ambilight worden opgeslagen;
+//2. Gebruik de filter-methode om de array te doorzoeken;
+//3. Geef de zoekcriteria mee: options.ambiLight === true;
+//4. Return de objecten;
+//5. Console log de variabele.
+
+const tvAmbilight = inventory.filter((typeAmbilight) => {
+    return typeAmbilight.options.ambiLight === true
```

Comment:
Dit kun je versimpelen:
```suggestion
    return typeAmbilight.options.ambiLight
```